spec/services/end_of_cycle/send_reject_by_default_reminder_to_providers_service_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe EndOfCycle::SendRejectByDefaultReminderToProvidersService do
+  describe '#call' do
+    context 'user was notified in a previous cycle', time: after_apply_deadline do
+      it 'creates a chased record' do
+        user = create(:provider_user, :with_provider, :with_notifications_enabled)
+        provider = user.providers.first
+        create(:chaser_sent,
+               chased: user,
```

Comment:
I wonder if a lot of this could be memoized? It would make it easier to read.

Reply:
I don't necessarily think it's easier to read if you have to keep scrolling back to the top to see what has been created, but I don't feel strongly about.